src/openms/source/SYSTEM/File.cpp

```diff
@@ -229,6 +229,12 @@ namespace OpenMS
     //add data dir in OpenMS data path
     directories.push_back(getOpenMSDataPath());
 
+    //add root dir
+    directories.push_back('/');
```

Comment:
this is a NOOP on Windows -- or might cause trouble - hard to say.
But why would you search in root?!
Maybe $HOME?